support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.jsx

```diff
@@ -104,36 +105,62 @@ const getDigitalImage = (isTop: boolean, countryGroupId: CountryGroupId) => {
   return <DigitalPackshot countryGroupId={countryGroupId} />;
 };
 
-const digital = (countryGroupId: CountryGroupId, priceCopy: PriceCopy, isTop: boolean): ProductCopy => ({
-  title: 'Digital Subscription',
-  subtitle: getDisplayPrice(countryGroupId, priceCopy.price),
-  description: countryGroupId === AUDCountries
-    ? 'The Guardian Editions app including Australia Weekend, Premium access to The Guardian Live app and ad-free reading on theguardian.com'
-    : 'The Guardian Editions app, Premium access to The Guardian Live app and ad-free reading on theguardian.com',
-  productImage: getDigitalImage(isTop, countryGroupId),
-  offer: priceCopy.discountCopy,
-  buttons: [{
-    ctaButtonText: 'Find out more',
-    link: digitalSubscriptionLanding(countryGroupId, false),
-    analyticsTracking: sendTrackingEventsOnClick({
-      id: 'digipack_cta',
-      product: 'DigitalPack',
-      ...(abTest && { abTest }),
-      componentType: 'ACQUISITIONS_BUTTON',
-    }),
-  },
-  {
-    ctaButtonText: 'See gift options',
-    link: digitalSubscriptionLanding(countryGroupId, true),
-    analyticsTracking: sendTrackingEventsOnClick({
-      id: 'digipack_cta_gift',
-      product: 'DigitalPack',
-      ...(abTest && { abTest }),
-      componentType: 'ACQUISITIONS_BUTTON',
-    }),
-    modifierClasses: '',
-  }],
-});
+const digital = (
+  countryGroupId: CountryGroupId,
+  priceCopy: PriceCopy,
+  isTop: boolean,
+  isVariant: boolean,
+): ProductCopy => {
+  const buttonsControl = [
+    {
+      ctaButtonText: 'Find out more',
+      link: digitalSubscriptionLanding(countryGroupId, false),
+      analyticsTracking: sendTrackingEventsOnClick({
+        id: 'digipack_cta',
+        product: 'DigitalPack',
+        ...(abTest && { abTest }),
+        componentType: 'ACQUISITIONS_BUTTON',
+      }),
+    },
+    {
+      ctaButtonText: 'See gift options',
+      link: digitalSubscriptionLanding(countryGroupId, true),
+      analyticsTracking: sendTrackingEventsOnClick({
+        id: 'digipack_cta_gift',
+        product: 'DigitalPack',
+        ...(abTest && { abTest }),
+        componentType: 'ACQUISITIONS_BUTTON',
+      }),
+      modifierClasses: '',
+    },
+  ];
+
+  const buttonsVariant = [
+    {
+      ctaButtonText: 'Subscribe now',
+      link: `${getOrigin()}/subscribe/digital/checkout?promoCode=DK0NT24WG&period=Monthly`,
```

Comment:
This is a bit of a hacky way of getting a link to the digital subscriptions checkout with the current offer applied. Seems acceptable for this test? Or is it too hacky?